pkg/clusteragent/autoscaling/workload/model/pod_autoscaler.go

```diff
@@ -148,16 +159,36 @@ func (p *PodAutoscalerInternal) UpdateFromSettings(podAutoscalerSpec *datadoghq.
 	p.horizontalEventsRetention = getHorizontalEventsRetention(podAutoscalerSpec.Policy, longestScalingRulePeriodAllowed)
 }
 
-// UpdateFromValues updates the PodAutoscalerInternal from a new scaling values
+// UpdateFromValues updates the PodAutoscalerInternal scaling values
 func (p *PodAutoscalerInternal) UpdateFromValues(scalingValues ScalingValues) {
 	p.scalingValues = scalingValues
 }
 
+// UpdateFromMainValues updates the PodAutoscalerInternal from new main scaling values
+func (p *PodAutoscalerInternal) UpdateFromMainValues(mainScalingValues ScalingValues) {
+	p.mainScalingValues = mainScalingValues
+}
+
+// UpdateFromLocalValues updates the PodAutoscalerInternal from new local scaling values
+func (p *PodAutoscalerInternal) UpdateFromLocalValues(fallbackScalingValues ScalingValues) {
+	p.fallbackScalingValues = fallbackScalingValues
+}
+
 // RemoveValues clears autoscaling values data from the PodAutoscalerInternal as we stopped autoscaling
 func (p *PodAutoscalerInternal) RemoveValues() {
 	p.scalingValues = ScalingValues{}
 }
 
+// RemoveMainValues clears main autoscaling values data from the PodAutoscalerInternal as we stopped autoscaling
+func (p *PodAutoscalerInternal) RemoveMainValues() {
```

Comment:
Is there any difference with `UpdateFromMainValues(ScalingValues{})`

Reply:
Good point, no difference, mostly added to stay consistent with the `RemoveValues()` function above but can remove these two